docs/reference/commandline/network_create.md

```diff
@@ -18,7 +18,7 @@ Create a network
 | `--ip-range`              | `stringSlice` |           | Allocate container ip from a sub-range                  |
 | `--ipam-driver`           | `string`      | `default` | IP Address Management Driver                            |
 | `--ipam-opt`              | `map`         | `map[]`   | Set IPAM driver specific options                        |
-| `--ipv6`                  |               |           | Enable IPv6 networking                                  |
+| `--ipv6`                  |               |           | Enable or disable IPv6 networking                       |
```

Comment:
Actually, this may have been a wrong suggestion; I mixed up things, and thought we changed the default, but the default is still the same, so adding the `or disable` may be confusing (at least currently).

I'll open a PR to revert this change.

Reply:
Oh, actually we did. Nevermind, I'm confused. Ignore me 🙈

Reply:
I think it's ok ... with `--default-network-opt=bridge=com.docker.network.enable_ipv6=true`, `network create --ipv6=false` would be needed to disable it for that network.

Reply:
Yes, it's indeed the combination of things that tripped me up for a bit. Let's keep it indeed, but we probably need more docs around this 😂